src/Symfony/Bundle/WebServerBundle/Resources/router.php

```diff
@@ -32,6 +32,10 @@
 
 $script = getenv('APP_FRONT_CONTROLLER') ?: 'index.php';
 
+if (false === getenv('APP_FRONT_CONTROLLER')) {
+    $script = $_ENV['APP_FRONT_CONTROLLER'];
```

Comment:
Not good, it should not squash the ternary above (when the env var doesn't exist, the value should remain 'index.php'). No need for calling getenv twice also

Reply:
changed